apps/studio/components/interfaces/Functions/EdgeFunctionSecrets/EdgeFunctionSecrets.tsx

```diff
@@ -15,11 +15,17 @@ import { Input } from 'ui-patterns/DataInputs/Input'
 import ConfirmationModal from 'ui-patterns/Dialogs/ConfirmationModal'
 import AddNewSecretForm from './AddNewSecretForm'
 import EdgeFunctionSecret from './EdgeFunctionSecret'
+import { EditSecretModal } from './EditSecretModal'
```

Comment:
Think this needs to be updated to match your new _EditSecretSheet_ name? I’m getting a build error:

```
Module not found: Can't resolve './EditSecretModal'
```

Reply:
There are a few references to EditSecretModal within EditSecretSheet still. E.g. `EditSecretModalProps`